### ISSUE TITLE
Icon Request: Fork

### ISSUE THREAD

**Issue by oskarkrawczyk** [feathericons/feather#283]
Name: Fork
Usage: Opposite to merge
Screenshots:

<img width="386" alt="screen shot 2018-01-18 at 15 23 14" src="https://user-images.githubusercontent.com/34213/35102552-869675e2-fc63-11e7-8bbd-41733a5a35e5.png">

**Comment by johnletey** [comment]
@oskarkrawczyk @locness3 I'm working on this now ... will have something up very shortly!

**Comment by johnletey** [comment]
@locness3 @colebemis @oskarkrawczyk How does this look? [Figma](https://www.figma.com/file/cQgvFjK5oh57wZ39b53HNt/git-fork)

<img width="395" alt="Screen Shot 2019-07-05 at 10 47 28 PM" src="https://user-images.githubusercontent.com/30328854/60742956-1f357600-9f78-11e9-9545-fc2c8b647ac9.png">

<img width="639" alt="Screen Shot 2019-07-05 at 10 55 39 PM" src="https://user-images.githubusercontent.com/30328854/60742957-1fce0c80-9f78-11e9-873f-3626ca2d4087.png">

**Comment by johnletey** [comment]
@ahtohbi4 What do you think?

**Comment by locness3** [comment]
Maybe making the rounded part like the `git-merge` icon ?

**Comment by ahtohbi4** [comment]
> @ahtohbi4 What do you think?

The familiar for everybody `git-fork` icon looks some differ. For example:

*Git*

<img width="97" alt="Снимок экрана 2019-07-06 в 13 33 28" src="https://user-images.githubusercontent.com/3686790/60757009-7b96a500-a00d-11e9-93c9-9ee042726619.png">

*GitLab*

<img width="116" alt="Снимок экрана 2019-07-06 в 13 33 10" src="https://user-images.githubusercontent.com/3686790/60757011-82251c80-a00d-11e9-9180-2b57098c9cf3.png">

*Codesandbox*

<img width="80" alt="Снимок экрана 2019-07-06 в 13 35 12" src="https://user-images.githubusercontent.com/3686790/60757014-88b39400-a00d-11e9-8733-31efa1b9c6cc.png">

So, I suppose that one should be something like this:

<img width="453" alt="Снимок экрана 2019-07-06 в 17 04 50" src="https://user-images.githubusercontent.com/3686790/60757216-388a0100-a010-11e9-8521-79940eb8fec0.png">

<img width="575" alt="Снимок экрана 2019-07-06 в 17 04 29" src="https://user-images.githubusercontent.com/3686790/60757217-3de74b80-a010-11e9-9469-e0d6c9d56664.png">

**Comment by locness3** [comment]
Great! maybe simplify the place where the top parts connect making a simple line, I think that it looks weird on small sizes

**Comment by johnletey** [comment]
That looks pretty good @ahtohbi4 ... but yes, I agree with @locness3

**Comment by ahtohbi4** [comment]
> Great! maybe simplify the place where the top parts connect making a simple line, I think that it looks weird on small sizes

Do you mean remove the circles on top?

**Comment by locness3** [comment]
No, I mean simplifying this part :
<img width="453" alt="60757216-388a0100-a010-11e9-8521-79940eb8fec0" src="https://user-images.githubusercontent.com/37651007/60789118-8d3b9200-a15e-11e9-942a-59cf4c11ab23.png">

**Comment by ahtohbi4** [comment]
Like this?

<img width="453" alt="Снимок экрана 2019-07-12 в 23 09 46" src="https://user-images.githubusercontent.com/3686790/61156320-bf9d1480-a4fb-11e9-91d2-68310af16671.png">

<img width="448" alt="Снимок экрана 2019-07-12 в 23 20 51" src="https://user-images.githubusercontent.com/3686790/61156330-c88de600-a4fb-11e9-8ae3-4221189d291c.png">

**Comment by locness3** [comment]
Yes!

**Comment by jgonera** [comment]
Could we get this merged?

**Comment by locness3** [comment]
No, but @ahtohbi4, if they are still alive, could submit this to @lucide-icons ;)
